src/coq_elpi_builtins.ml

```diff
@@ -1197,8 +1199,10 @@ It's a fatal error if Name cannot be located.|})),
     let mind, indbo as ind = Inductive.lookup_mind_specif env i in
     let lno = mind.mind_nparams in
     let luno = mind.mind_nparams_rec in
+    let ntyps = mind.mind_ntypes in
     let ty = if_keep ty (fun () ->
       Inductive.type_of_constructor (kon,Univ.Instance.empty) ind
+      |> Inductive.abstract_constructor_type_relatively_to_inductive_types_context ntyps (fst i)
```

Comment:
I don't understand this change, it makes the type point to rels which don't exists. The old API was returning a closed term (see also the `COut(closed_ground_term,"Ty",` line above). But maybe I've misunderstood this line?

Reply:
You're right, nothing needs to be changed here. I got confused. I'm going to fix it.